spec/spec.md

```diff
@@ -0,0 +1,180 @@
+
+- The files in the `.reuse/` directory in the root of the project.
+
+<!-- TODO: The files inside of a git submodule -->
+
+There are various ways to associate copyright and licensing information with a
+file.
+
+### Comment headers
+
+Each plain text file that can contain comments should contain comments at the
+top of the file (comment header) that declare that file's copyright and
+licensing information.
+
+If a file is not a plain text file or does not permit the inclusion of
+comments, the comment header that declares the file's copyright and licensing
+information should be in an adjacent file of the same name with the
+additional extension `.license` (example: `cat.jpg.license` if the original
+file is `cat.jpg`).
+
+The comment header must contain one or more `SPDX-Copyright` tags, and one or
+more `SPDX-License-Information` tags. A tag is followed by a colon, followed by
+a text value, and terminated by a newline.
+
+The `SPDX-Copyright` tag must be followed by a copyright notice as described
+in <!-- FIXME: Refer to an internal section -->.
+
+Instead of the `SPDX-Copyright` tag, the symbols `Copyright`, `Copyright (c)`,
+`Copyright (C)`, `Copyright ©` or `©` may be used.
+
+The `SPDX-License-Identifier` tag must be followed by a valid SPDX License
+Expression describing the licensing of the file (example:
+`SPDX-License-Identifier: GPL-3.0-or-later OR Apache-2.0`). If separate sections
+of the file are licensed differently, a different `SPDX-License-Identifier` tag
+should be included for each section.
+
+An example of a comment header:
+
+```
+# SPDX-Copyright: 2016, 2018-2019 Jane Doe <jane@example.com>
+# SPDX-Copyright: 2019 Example Company
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+```
+
+### DEP5
+
+Copyright and licensing information may be associated with a file through a DEP5
+file. The intended use case of this method is large directories where including
+a comment header in each file (or in `.license` companion files) is impossible
+or undesirable.
+
+The DEP5 file must be in `.reuse/dep5` from the root of the project.
+
+<!-- TODO: How to do multiple licenses without AND and OR. -->
+
+The `License` tag must be followed by a valid SPDX License Expression describing
+the licensing of the associated files.
+
+The `Copyright` tag must be followed by a copyright notice as described in <!--
+FIXME: Refer to an internal section -->.
+
+An example of a DEP5 file:
+
+```
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Project
+Upstream-Contact: Jane Doe <jane@example.com>
+Source: https://example.com/jane/project
+
+Files: po/*
+Copyright: 2019 Translation Company
+License: GPL-3.0-or-later
+```
+
+## Format of copyright notices
+
+A copyright notice must start with the copyright symbol (©), the word
+"Copyright", or an equivalent tag, followed by the year(s) of publication, the
+name of the copyright holder, and the e-mail address of the copyright holder.
```

Comment:
```suggestion
name of the copyright holder, and a contact of the copyright holder. In case the copyright holder is a natural person, their e-mail address would be a good contact; in case the copyright holder is a legal entity, its website would be a good point of contact.
```

Reply:
Agree with the change from e-mail address to contact. I would rephrase the second sentence to be shorter, however, and include the word "example".

Reply:
Sounds like a good change to me.

Reply:
Fixed in 9956e990765be298524249146a0eb9140ecf7590